pybleau/app/io/tests/test_plotting_roundtrip.py

```diff
@@ -106,19 +104,26 @@ def test_round_trip_df_plotter_heatmap_types(self):
         config7.z_col_name = "Col_3"
         desc7 = PlotDescriptor(x_col_name="Col_1", y_col_name="Col_2",
                                z_col_name="Col_3",
-                               plot_config=config7, plot_title="Plot 6")
+                               plot_config=config7, plot_title="Plot 7")
 
-        analyzer2 = DataFrameAnalyzer(source_df=TEST_DF2)
-
-        plot_manager2 = DataFramePlotManager(
-            contained_plots=[desc7],
-            data_source=TEST_DF2, source_analyzer=analyzer2
-        )
-
-        self.assert_roundtrip_identical(plot_manager2, ignore=self.ignore)
+        self.assert_df_plotter_roundtrip(desc7)
 
     # Support methods ---------------------------------------------------------
 
+    def assert_df_plotter_roundtrip(self, desc, adtl_ignore=()):
+        adtl_ignore = list(adtl_ignore)
+        source_df = desc.plot_config.data_source
+        analyzer = DataFrameAnalyzer(source_df=source_df)
+
+        options = [[desc], [desc.plot_config], [desc.plot_config] * 3]
```

Comment:
This seems like overkill. The intention is understood (we should be able to roundtrip `PlotDescriptors`, `Configurators`, and a list of `Configurators`. We should really be relying on other tests that ensure the `DataFramePlotManager` knows how to handle each of these three different types for its `contained_plots` argument, though. This current test is just making sure the serialization is correct *after* the plot manager deciphers the input argument. 
Just a thought.

Reply:
Agreed. Simplified.